backstage/backstage
# [Search] Add search components to storybook

## Hey, I just made a Pull Request!

This PR adds some of our search components, used to compose the [SearchPage](https://github.com/backstage/backstage/blob/master/packages/app/src/components/search/SearchPage.tsx) in the App, to storybook. The components are added under its own "Search" section.

<img width="1439" alt="Screen Shot 2021-07-09 at 12 16 13 PM" src="https://user-images.githubusercontent.com/25153114/125064033-7e22ae00-e0b0-11eb-82e8-04d0005fdf60.png">

###  How to test:

1. Run Storybook locally by following the ["Running Locally"](https://backstage.io/docs/dls/contributing-to-storybook#running-locally) instructions.
2. You should see the search section with 4 stories under it, `DefaultResultListItem`, `SearchBar`, `SearchFilter` and `SearchResult`.

`SearchContextProvider` uses `searchApiRef` for the `searchApi`, which we don't need for the component examples, that's why `SearchContext.Provider` is used instead. 

#### :heavy_check_mark: Checklist
